### ISSUE TITLE
CI job deploy-app-store fails: The Info.plist in the package must contain the CFBundleShortVersionString key

### ISSUE THREAD

**Issue by bafu** [numbersprotocol/capture-lite#1891]
## Description
In the [Attemp 1](https://github.com/numbersprotocol/capture-lite/runs/7636096211?check_suite_focus=true) of deploy-app-store,

```
/usr/bin/xcrun altool --output-format xml --upload-app --file build/App.ipa --type ios --apiKey *** --apiIssuer ***
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>os-version</key>
	<string>11.6.8</string>
	<key>product-errors</key>
	<array>
		<dict>
			<key>code</key>
			<integer>-27002</integer>
			<key>message</key>
			<string>The Info.plist in the package must contain the CFBundleShortVersionString key.</string>
			<key>userInfo</key>
			<dict>
				<key>NSLocalizedDescription</key>
				<string>The Info.plist in the package must contain the CFBundleShortVersionString key.</string>
				<key>NSLocalizedFailureReason</key>
				<string>Unable to validate your application.</string>
			</dict>
		</dict>
	</array>
	<key>tool-path</key>
	<string>/Applications/Xcode_13.2.1.app/Contents/SharedFrameworks/ContentDeliveryServices.framework/Versions/A/Frameworks/AppStoreService.framework</string>
	<key>tool-version</key>
	<string>4.071.1221</string>
</dict>
</plist>


Error: The process '/usr/bin/xcrun' failed with exit code 1
```

## Logs
In the step "Build iOS Xcode Archive"

Successful 0.59.1

```
Build settings from command line:
    CURRENT_PROJECT_VERSION = 20220622034142
    MARKETING_VERSION = 0.59.1
    SDKROOT = iphoneos15.2
```

Failed 0.62.0

```
Build settings from command line:
    CURRENT_PROJECT_VERSION = 20220802164215
    MARKETING_VERSION = 
    SDKROOT = iphoneos15.2
```

## Environment
1. capture-lite: 0.62.0

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201016280880500/1202711468921279) by [Unito](https://www.unito.io)

**Comment by bafu** [comment]
We have changed the trigger event from branch to tag. When pushing a version tag, the module version is likely be the same in current and prior commits.

If the action `EndBug/version-check@v2` does not find a change, `outputs.version` will be empty.

**Comment by bafu** [comment]
Use `GITHUB_REF` to extract the tag name instead of using `EndBug/version-check@v2`:

```
      - name: Get release version
        id: version_check
        run: echo ::set-output name=version::${GITHUB_REF/refs\/tags\//}
```

**Comment by bafu** [comment]
Fixed by commit `d2aeadf`
